docs/readmes/lte/deploy_install.md

```diff
@@ -94,7 +92,7 @@ journalctl -fu agw_installation
 When you see "AGW installation is done.", it means that your installation is complete. You can make sure magma is running by executing
 
 ```bash
-service magma@* status
+service 'magma@*' status
```

Comment:
Why is this needed? It seems to be written without the quote marks throughout the repo.

Reply:
Because * will not be interpreted as a string literal. This will evaluate to different things depending on the directory where the command is being run. Adding single quotes ensures that the value is the same regardless of the context. It can't do damage because the shell will interpret them as metachars and remove them before passing in to the service params.

Reply:
Okay, thanks for the clarification!